provisioning/roles/kuma/files/apache2/all-servers.conf

```diff
@@ -3,15 +3,6 @@
 # Our servers symlink to this file. Do not move it without first coordinating
 # with WebOps.
 
-<Location /static/fonts>
-    # Only enable CORS for fonts when the request is coming from an approved domain
-    Header unset Access-Control-Allow-Origin
-    SetEnvIf Origin "https?://(.*\.(mozilla|allizom)\.(com|org|net))" CORS=$0
-    SetEnvIf Origin "https?://(mdn\.mozillademos\.org)" CORS=$0
-    SetEnvIf Origin "https?://(kuma\.readthedocs\.org)" CORS=$0
-    Header set Access-Control-Allow-Origin %{CORS}e env=CORS
```

Comment:
Here's some more explanation why it does that: https://github.com/evansd/whitenoise/blob/0657e7aea986fa652bd04d6ae8468a0e1129d634/whitenoise/base.py#L90-L95